scripts/Copyright-Checker/copyright-checker.sh

```diff
@@ -156,15 +171,11 @@ function perform_copyright_check() {
     is_copyright_valid "${file}"
     local is_valid="${?}"
 
-    is_ui_example "${file}"
-    local is_example="${?}"
-
-    # invert the copyright-check for UI examples, because we
-    # don't want them to have too much content.
-    if ([ 0 -eq ${is_example} ] && [ 0 -eq ${is_valid} ]) ||
-      ([ 1 -eq ${is_example} ] && [ 1 -eq ${is_valid} ]); then
+    if [ 1 -eq ${is_valid} ]; then
       printf "copyright is not as expected in %s\n" "${file}"
       exit_status=1
+    elif [ 2 -eq ${is_valid} ]; then
+      printf "copyright is not allowed in %s\n" "${file}"
```

Comment:
```suggestion
      printf "copyright is not allowed in %s\n" "${file}"
      exit_status=2
```
Missing exist status. Or should this error not cause the whole script to return an error?
You can also simply call `exit_status="$is_valid"` and remove both lines from the `if` blocks (like I suggested in the previous comment with the `case` example)